config.js

```diff
@@ -56,26 +57,26 @@ const config = {
   assetValueDecimalPlaces: 2,
   transactionUrls: {
     BTC: {
-      Mainnet: 'https://live.blockcypher.com/btc/tx',
-      Testnet: 'https://live.blockcypher.com/btc-testnet/tx',
+      Mainnet: 'https://live.blockcypher.com/btc/txs',
+      Testnet: 'https://live.blockcypher.com/btc-testnet/txs',
     },
     RBTC: {
-      Mainnet: 'https://explorer.rsk.co/tx',
-      Testnet: 'https://explorer.testnet.rsk.co/tx',
+      Mainnet: 'https://explorer.rsk.co/txs',
```

Comment:
I dont have this change

Reply:
Let's review it in another PR. I reverted the change since I don't need it either